src/controllers/TravelController.js

```diff
@@ -55,3 +55,91 @@ export async function createTravel(req, res) {
     return res.status(400).json({ error: `Bad request: ${e?.message}` })
   }
 }
+
+export async function addTravelStep(req, res) {
+  try {
+    const { user: { id: userId } } = req
+
+    const {
+      travelID,
```

Comment:
`travelId` pour rester cohérent avec les autres routes
```suggestion
      travelId,
```